src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/ArmBase.PlatformNotSupported.cs

```diff
@@ -11,7 +11,12 @@ namespace System.Runtime.Intrinsics.Arm
     /// This class provides access to the ARM base hardware instructions via intrinsics
     /// </summary>
     [CLSCompliant(false)]
-    public abstract class ArmBase
+#if USE_INTERNAL_ACCESSIBILITY
```

Comment:
I think it would make sense to remove the CLSCompliant attribute when `USE_INTERNAL_ACCESSIBILITY` is enabled. Getting build warnings which I have to explicitly disable.

Reply:
Ah yes. Will submit a small PR for this shortly